terraform/scripts/update_geoip.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# update_geoip.sh - Update the MaxMind GeoIP2 database on a target system over
+#  Secure Shell (SSH).  This is done by downloading and verifying the database
+#  locally before using Secure Copy (SCP) to copy the database archive to the
+#  target system and extracting it to the appropriate path on the target system
+#  using SSH.
```

Comment:
This single space indent is annoying, since it's not visually obvious and looks like a typo.  Can you make it two spaces (if it was intended) or get rid of it altogether (if it was not)?